scripts/run-integration-tests.sh

```diff
@@ -16,6 +16,7 @@ K8S_VERSION=${K8S_VERSION:-1.14.6}
 PROVISION=${PROVISION:-true}
 DEPROVISION=${DEPROVISION:-true}
 BUILD=${BUILD:-true}
+RUN_CONFORMANCE=${RUN_CONFORMANCE:-true}
```

Comment:
default this to "false" please, to keep existing behaviour of the run-integration-tests.sh script.

Reply:
Thanks for the review Jay. As discussed offline, only reason was to run conformance test on PRs until we have fully qualified integration tests. I will set it to true through env variable